app/controllers/api/v2/rh_cloud/inventory_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V2
+    module RhCloud
+      class InventoryController < ::Api::V2::BaseController
+        include ::Api::Version2
+        include InventoryUpload::ReportActions
+        include InventoryUpload::TaskActions
+
+        api :GET, "/organizations/:organization_id/rh_cloud/report", N_("Download latest report")
+        def download_file
+          filename, file = report_file(params[:organization_id])
+
+          send_file file, disposition: 'attachment', filename: filename
```

Comment:
In case there is no report and `report_file` raises `ReportMissingError` shouldn't we catch it return 404 instead of just crashing with 500 (or at least that's what I think would happen)

Reply:
Corrected